rapid-threat-command/README.md

```diff
@@ -2,7 +2,7 @@
 Parameter                           | Name | Default Value | Type | Required (True/False) | Description
 ---                                 | --- | --- | --- |--- |---
 hostname                            | Host Name | https://api.ti.insight.rapid7.com | String | True | IP or URL for the instance.
```

Comment:
@Benimanela same here, there is any IP? it is cloud-based platform.

Reply:
done